app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -1652,3 +1647,30 @@ msgstr "Mae %(errorcount)s o broblemau gyda'r dudalen hon"
 msgid "There are %(errorcount)s problems with your answer"
 msgstr "Mae %(errorcount)s o broblemau gyda'ch ateb"
 
+#~ msgid "You have selected an address in Northern Ireland."
```

Comment:
These lines in po files can be removed - the po files keep any translations that had been previously entered but are no longer in the pot file. They can be removed to keep the files tidy, but it's not a problem to leave them behind

Reply:
Removed them, then changed my mind and left them. Will remove them next time we change some content, which I expect won't be that long.